upgrade.py

```diff
@@ -487,6 +524,7 @@ def buildRestoreList(self, src_base):
         return restore_list
 
     completeUpgradeArgs = ['mounts', 'installation-to-overwrite', 'primary-disk', 'backup-partnum', 'logs-partnum', 'net-admin-interface', 'net-admin-bridge', 'net-admin-configuration']
+
```

Comment:
Not sure if this was intentional or accidental but none of the other functions have a blank line between `funcArgs = ...` and `def func(...`

Reply:
It's accidental :(